dot/network/host.go

```diff
@@ -394,3 +394,17 @@ func (h *host) protocols() []string {
 func (h *host) closePeer(peer peer.ID) error {
 	return h.h.Network().ClosePeer(peer)
 }
+
+func (h *host) closeProtocolStream(pID protocol.ID, p peer.ID) {
+	connToPeer := h.h.Network().ConnsToPeer(p)
+	for _, c := range connToPeer {
+		for _, st := range c.GetStreams() {
+			if st.Protocol() == pID {
+				err := st.Close()
+				if err != nil {
+					logger.Tracef("Failed to close stream", "protocol", pID, "error", err)
```

Comment:
Also this will have to change when merging development with the newer log package (`internal/log`).

Reply:
Done